### DISCUSSION TITLE
cpu usage high at doc page

### DISCUSSION THREAD

**Discussion by limi58** [vercel/next.js#42139] [Show and tell]
nextjs doc page has a button on fixed header, name "Next.js Conf"

it has blur and animation style cause MacBook Pro highly cpu usage, watching doc then the fan is noisy, can remove it animation?

**Comment by akupar** [comment]
Thanks for pointing out the blur effect! 

I have the same issue trying to read the tutorial at https://nextjs.org/learn/foundations/about-nextjs. With Firefox the CPU usage goes to 130 % and with Chrome there's two processes with 35 % CPU usage. Granted, my computer is pretty old.

For anyone having the same issue, open the browser console and do `document.querySelector('.page-container header').remove(); `, and you can hear the birds sing again.
